app/controllers/api/v1/admin/users_controller.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
```

Comment:
What would this controller be in charge of?

Reply:
I'm not sure, I spoke to @farhatahmad and he said to make one. Could you explain Ahmad?

Reply:
What about the `users_controller#index`?
This way we're creating a differentiating between all roles and an Admin.
Admins now have their own API when actually Admin is a Role just like `User` and `Guest` but with absolute permissions.
If we're still designing GL3 to be permission based and since it's now represented as an Restful API I think it makes more sense to have our controllers subject based like Users, Recordings, ... and make the proper restrictions so that accessing certain endpoints or accessing certain data is controlled.

Reply:
If a certain area of the API encapsulates a lot of complexity then for simplicity sake we can extract it into its own controller, Is that what you intended @farhatahmad ?